tests/test_dataframes.py

```diff
@@ -64,3 +65,30 @@ def test_pandas_extension(ocx, api_data):
         "204441",
         "1783270",
     ]
+
+# Current, there's a bug in pandas where these types of operations don't carryover
```

Comment:
do you a have a link to a GH issue for it?

Reply:
I'm not sure if @clausmith has opened it yet. @clausmith please add once open on https://github.com/pandas-dev/pandas

Reply:
can we create it before merging?

Reply:
Yes I think so. There are about half a dozen other issues open already about this functionality, and there's some discussion about deprecating it completely. I'm just trying to figure out whether there's already an issue open specifically about _our_ bug.